src/js/form-builder.js

```diff
@@ -317,9 +317,7 @@ const FormBuilder = function(opts, element) {
       field = Object.assign({}, $field)
     }
 
-    if (!field.name) {
```

Comment:
Without this check, what happens when saved data is loaded? will it replace the name?

Reply:
Actually, I found `utils.nameAttr()` output to be the same as `field.name` in most cases, maybe I'll look into more. Because I thought the `FormBuilder` generated that value automatically.

Reply:
It does generate it if one isn't set. My concern was that if an existing form was loaded, the saved value could be overwritten. If a user sets a custom name, that custom value would be overwritten.

Reply:
Cool, i've check the field name inside name utility function